macros/src/lib.rs

```diff
@@ -48,7 +48,7 @@ pub fn derive_bundle(input: TokenStream) -> TokenStream {
 /// This is an extension macro for bundles which allow them to be cloned, and
 /// subsequently used in `EntityBuilderClone::add_bundle`.
 ///
-/// Requires that all fields of the struct implement clone.
+/// Requires that all fields of the struct implement Clone.
```

Comment:
Because `Clone` is a resolveable identifier, it should be referenced with an intra-doc link, using the syntax [\`Clone\`].

Reply:
Thank you. Learning a lot of Rust here.